src/libs/ReportActionsUtils.ts

```diff
@@ -374,14 +374,21 @@ function shouldIgnoreGap(currentReportAction: ReportAction | undefined, nextRepo
  * Returns a sorted and filtered list of report actions from a report and it's associated child
  * transaction thread report in order to correctly display reportActions from both reports in the one-transaction report view.
  */
-function getCombinedReportActions(reportActions: ReportAction[], transactionThreadReportActions: ReportAction[], reportID?: string): ReportAction[] {
-    if (isEmptyObject(transactionThreadReportActions)) {
+function getCombinedReportActions(
+    reportActions: ReportAction[],
+    transactionThreadReportActions: ReportAction[],
+    reportID?: string,
+    transactionThreadReportID?: string | null,
+): ReportAction[] {
+    // There is a chance that transactionThreadReportActions is not loaded yet.
+    // We only check for emptiness when there are multiple IOUs,
+    // this will prevent the sub-report preview from being displayed when there is only one IOU.
```

Comment:
Do you think we don't need this comment now?

Reply:
@Pujan92 I believe the comment is helpful as it provides a rationale for the necessity of checking for an empty `transactionThreadReportID`. Perhaps we could consider rephrasing it for clarity, any suggestion?

Reply:
> // There is a chance that transactionThreadReportActions is not loaded yet.

I thought now that we have the `transactionThreadReportID` we don't need to rely on `transactionThreadReportActions` for that condition. If we go with that then we can omit this comment bcoz the function description is enough to convey. WDYT?